src/components/header/ProfileThreadTracingMarkerOverview.js

```diff
@@ -15,8 +15,11 @@ export default connect((state, props) => {
   const { threadIndex } = props;
   const selectors = selectorsForThread(threadIndex);
   const selectedThread = getSelectedThreadIndex(state);
+  const intervalMarkers = selectors
+    .getRangeSelectionFilteredTracingMarkers(state)
+    .filter(tm => tm.name !== 'GCMajor');
```

Comment:
not sure this is the right place, but I thought it was :) Maybe we'd want to make this more generic in case we want to hide more markers here ?